pkg/dir.props

```diff
@@ -2,6 +2,10 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\dir.props" />
 
+  <PropertyGroup>
```

Comment:
Could you please add a comment here explaining the property? Something along the lines of: "Used to determine if we should ensure we build some packages only once across multiple official build legs. For offline builds we still set OfficialBuildId but we need to build all the package for a single leg so we also take DotNetBuildOffline into account."